services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -634,6 +632,10 @@ private void sendAdminMessageAndWaitForConsumed(String clusterName, String store
   @Override
   public void deleteValueSchemas(String clusterName, String storeName, Set<Integer> unusedValueSchemaIds) {
     Set<Integer> inuseValueSchemaIds = getInUseValueSchemaIds(clusterName, storeName);
+    if (inuseValueSchemaIds.isEmpty()) {
+      LOGGER.error("For store {} cannot find in-use schema ids.", storeName);
```

Comment:
Should this log be always in error level?
Such as if there is no active store versions, it is expected to have empty in-used schemas, right?
Shall we only log this in error level only the store has some active versions?

Reply:
Sure, we can add a check for active versions